extensions/control-plane/transfer/transfer-data-plane-signaling/src/test/java/org/eclipse/edc/connector/transfer/dataplane/flow/DataPlaneSignalingFlowControllerTest.java

```diff
@@ -350,4 +315,63 @@ private TransferProcess.Builder transferProcessBuilder() {
                 .counterPartyAddress("test.connector.address")
                 .dataDestination(DataAddress.Builder.newInstance().type("test").build());
     }
+
+    @Nested
+    class Suspend {
```

Comment:
I was late for review, but why put tests under private methods? Doesn't look correct

Reply:
which tests?